integrations/dashly/lib/index.js

```diff
@@ -0,0 +1,223 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+var integration = require('@segment/analytics.js-integration');
+
+/**
+ * Expose `Dashly` integration.
+ */
+
+var Dashly = (module.exports = integration('Dashly')
+  .assumesPageview()
+  .global('dashly')
+  .global('dashlyasync')
+  .option('apiKey', null)
+  .tag('<script src="//cdn.dashly.app/api.min.js"></script>'));
+
+/**
+ * Initialize.
+ *
+ * https://dashly.io/developers/tuning/install/
+ *
+ * @api public
+ */
+
+Dashly.prototype.initialize = function() {
+  var apiKey = this.options.apiKey;
+
+  window.dashly = {};
+  window.dashlyasync = [];
+
+  var m = [
+    'addCallback',
+    'auth',
+    'connect',
+    'identify',
+    'onReady',
+    'open',
+    'removeCallback',
+    'track',
+    'trackMessageInteraction'
+  ];
+
+  for (var i = 0; i < m.length; i++) {
+    window.dashly[m[i]] = Build(m[i]);
+  }
+
+  window.dashly.connect(apiKey);
+
+  this.load(this.ready);
+
+  function Build(name) {
+    return function() {
+      window.dashlyasync.push(name, arguments);
+    };
+  }
+};
+
+/**
+ * Loaded?
+ *
+ * @api private
+ * @return {boolean}
+ */
+
+Dashly.prototype.loaded = function() {
+  return !!(window.dashly && window.dashly.connected);
+};
+
+/**
+ * Identify.
+ *
+ * @api public
+ * @param {Identify} identifyFacade
+ */
+
+Dashly.prototype.identify = function(identifyFacade) {
```

Comment:
Is this called "identifyFacade" officially? From usage it's an integration object, not facade 🤔